compiler/codegen/symbol.cpp

```diff
@@ -786,6 +786,24 @@ static Type* getArgSymbolCodegenType(ArgSymbol* arg) {
   return useType;
 }
 
+// Alter the C code type to avoid _ref_, c_ptr_ prefixes and instead use a
+// type that C understands without additional information (e.g. instead of
+// _ref_int64_t, just int64_t *).  But only for exported symbols like the
+// return type of exported functions, or arguments of those functions.
```

Comment:
I believe this can also apply to DDATA, as long as it isn't wide. Might be worth a TODO comment.